controllers/BooksIndexController.js

```diff
@@ -0,0 +1,21 @@
+angular
+  .module('books')
+  .controller('BooksIndexController', BooksIndexController);
+
+BooksIndexController.$inject = ['$http'];
+
+function BooksIndexController ($http) {
+  var vm = this;
+
+  $http({
+    method: 'GET',
+    url: 'https://super-crud.herokuapp.com/books'
+  }).then(function successCallback(response) {
+    console.log("success")
```

Comment:
Don't need the success console log because you have it logging the response.data.